tools/marvin/marvin/config/test_data.py

```diff
@@ -791,6 +791,8 @@
         "displaytext": "xs",
         "name": "xs",
         "passwordenabled": False,
+        "ostype": "CentOS 5.6 (64-bit)"
```

Comment:
Please don't use hard-coded values. Will this work when template os type changes in the setup?